examples/README.md

```diff
@@ -1,23 +1,36 @@
 # Examples
 
-Here you can find a bunch of examples of workflows / functions written for the
-Edgeless MVP platform. 
+Here you can find examples of EDGELESS workflows using the functions shipped
+with the repository in [functions](../functions/) and showcasing some of the key
+features provided by the platform.
 
 - `container`: shows how to deploy a mixed workflow consisting of a function instance running in WASM, another as a container, and a resource
+- `esp32_resources`: shows how to use a `scd30-sensor` resource
```

Comment:
also uses the epaper display resource (but that can be fixed once the other pr is merged)